apis/Google.Cloud.Firestore/Google.Cloud.Firestore/Google.Cloud.Firestore.csproj

```diff
@@ -16,7 +16,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Update="Filter.*.cs">
-        <DependentUpon>Filter.cs</DependentUpon>
+      <DependentUpon>Filter.cs</DependentUpon>
```

Comment:
Not sure what this is doing here. This same change was already merged by another one of these release PRs where I didn't spot this in time. And it seems to be harmless. @jskeet I'll wait for a bit before approving the rest of the PRs, let me know if you have any objections.

Reply:
Hmm... I suspect this was incorrect in Hemansh's PR, and then got fixed by regenerating projects. Yes, it's harmless.